pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -121,19 +119,17 @@ async def run(self):
                 # Create a PR to pin microshift build
                 assembly_basis = get_assembly_basis(releases_config, self.assembly)
                 if assembly_basis.brew_event:
-                    pr = await self._create_or_update_pull_request(nvrs)
+                    pr = await self._update_release_in_pr(nvrs)
+
+                # Sends a slack message
+                if slack_client:
+                    message = f"Assembly definition has been updated with {pr}"
+                    if assembly_type is AssemblyTypes.PREVIEW:
+                        message += "\n This is an EC release. Please run <https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/aos-cd-builds/job/build%252Fmicroshift_sync/|microshift_sync> with `UPDATE_PUB_MIRROR` checked."
+                    await slack_client.say(message, slack_thread)
+
+            self._attach_builds(group_config.get("advisories").get("microshift"))
```

Comment:
This job could be run to build microshift nightlies that are never intended to release.